capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module

```diff
@@ -293,3 +293,19 @@ function c4m_content_group_submit_redirect(&$form, &$form_state) {
   // Redirect a user to homepage.
   $form_state['redirect'] = url('', array('absolute' => TRUE));
 }
+
+/**
+ * Implements hook_custom_theme().
+ *
+ * Set Kapablo theme for group creation page.
+ */
+function c4m_content_group_menu_alter(&$items) {
```

Comment:
I don't believe this is the solution:

* The **settings for the Admin theme** already state that all content create/edit/delete forms **should be rendered in the Default theme (Kapablo)**.
* This **works for all node types** (including projects) **except Groups**.
* There is probably an overwrite already of this behaviour that breaks the proper theme for groups.
* The **group edit form does use the Kapablo theme**, it's only the create that has a problem.